scripts/qa_crowbarsetup.sh

```diff
@@ -1893,6 +1924,16 @@ function onadmin_post_allocate()
 {
     pre_hook $FUNCNAME
 
+    # for testing the Manila generic driver, the nodes with the m-shr service
+    # and the manila-service VM need an IP from the same subnet. So add all
+    # current nodes to the subnet
+    if iscloudver 6plus; then
+        cmachines=$(get_all_nodes)
```

Comment:
Hm. I think it doesn't matter. The important thing is that the nodes with the ```manila-share``` service running get an IP from the network. But I can change it if you want...